src/Bicep.LangServer.UnitTests/BicepCompletionProviderTests.cs

```diff
@@ -72,7 +72,7 @@ public void DeclarationSnippetsShouldBeValid()
             foreach (var (detail, completion) in snippetsByDetail)
             {
                 // validate snippet
-                var snippet = new Snippet(completion.TextEdit!.NewText);
+                var snippet = new Snippet(completion.TextEdit!.TextEdit!.NewText);
```

Comment:
>TextEdit [](start = 63, length = 8)

This fun is due to the introduction of `InsertReplaceTextEdit`. I haven't played with it yet, though.